google/cloud/security/iam/explain/utils.py

```diff
@@ -13,3 +13,34 @@
 # limitations under the License.
 
 """ Explain utilities """
+import datetime
+
+# pylint: disable=missing-type-doc,missing-return-type-doc,missing-return-doc
+# pylint: disable=missing-param-doc,missing-raises-doc
+
+def _unified_time(time_str):
+    """ Return utc time, dealing with or without time zone info"""
```

Comment:
ditto: please document the parameters and return

You can see examples here:
http://www.sphinx-doc.org/en/stable/ext/example_google.html